app.js

```diff
@@ -8,6 +8,11 @@ const bodyParser = require('body-parser');
 const path = require('path');
 
 const solidityAPI = require('./solidity/facades/backend');
+const Shape = require('./solidity/facades/shape');
+const web3 = require('./solidity/web3/rinkeby');
+solidityAPI.useWeb3(web3);
+Shape.useWeb3(web3);
```

Comment:
I believe this line is unnecessary, since solidityAPI.useWeb3 forwards the given web3 to Shape.useWeb3.